### ISSUE TITLE
Support for AWS Config Conformance Packs

### ISSUE THREAD

**Issue by lorengordon** [hashicorp/terraform-provider-aws#11098]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add resource for managing AWS Config Conformance Packs.

### New or Affected Resource(s)

* aws_config_conformance_pack
* aws_config_organization_conformance_pack

### Potential Terraform Configuration

```hcl
resource "aws_config_conformance_pack" "this" {
  name                   = string
  delivery_s3_bucket     = string
  delivery_s3_key_prefix = string
  template_body          = string
  template_s3_uri        = string

  input_parameters {
    name  = string
    value = string
  }
```

### References

* API: https://docs.aws.amazon.com/config/latest/APIReference/API_PutConformancePack.html
* User Guide: https://docs.aws.amazon.com/config/latest/developerguide/conformance-packs.html
* Blog Post: https://aws.amazon.com/blogs/mt/introducing-aws-config-conformance-packs/

**Comment by breathingdust** [comment]
Hi all! :wave: Just wanted to direct you to our public [roadmap](https://github.com/hashicorp/terraform-provider-aws/blob/master/ROADMAP.md) for this quarter (Nov-Jan) in which this item has been mentioned.

Due to the significant community interest in support for this feature, we will be looking at merging existing contributions soon. 

We appreciate all the contributions and feedback thus far.

Look out for support in the provider soon!

**Comment by anGie44** [comment]
Hi @lorengordon et al. 👋 Support for the `aws_config_conformance_pack` resource has been merged and will release with version 3.28.0 of the Terraform AWS Provider.

**Comment by parente** [comment]
🎉 https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/config_conformance_pack

v3.28.0 of the Terraform AWS Provider
